src/tools/perf/perftest_daemon.c

```diff
@@ -409,18 +448,10 @@ ucp_perf_daemon_peer_tx_handler(void *arg, const void *header,
 
 static void ucp_perf_daemon_cleanup(ucp_perf_daemon_context_t *ctx)
 {
-    if (NULL != ctx->send_memh) {
-        /* coverity[check_return] */
-        ucp_mem_unmap(ctx->context, ctx->send_memh);
-    }
+    ucs_trace_data("cleaning up daemon");
```

Comment:
maybe just `trace` because it is not on a data path?

Reply:
ok, changed here and in ucp_perf_daemon_cleanup_connections